s4e-backend/src/main/java/pl/cyfronet/s4e/listener/GroupListener.java

```diff
@@ -27,17 +28,18 @@
     private final MessageSource messageSource;
     private final TemplateEngine templateEngine;
     private final MailService mailService;
+    private final GroupService groupService;
 
     @Value("${mail.urlDomain}")
     private String urlDomain;
 
     @Async
     @EventListener
+    @Transactional(readOnly = true)
```

Comment:
Why these `@Transactional`?

Reply:
because institution in group is lazy and i use here normal method from repo, without query